google/ggrc-core
# GGRC-6796 Endpoint for unmapping objects

# Issue description
Create endpoint which will unmap objects by object type and object id.

# Steps to test the changes

1. Create any 2 objects of mappable type
2. Create 1 or more relationships between created objects
3. Send POST request to "/api/relationships/unmap" endpoint
3.1 `first_object_type` - type of first object
3.2 `first_object_id` - ID of first object
3.3 `second_object_type` - type of second object
3.4 `second_object_id` - ID of second object
4. All relationships between objects should be deleted

# Solution description

1. Endpoint searches all relationships with `source` as first object and `destination` as second object, or `source` as second object and `destination` as first object.
2. For each found relationship:
2.1. Delete relationship in session
2.2. Sends `model_deleted` signal
2.3. Create revision
2.4. Commit session
2.5. Sends `model_deleted_after_commit` signal

# Sanity checklist




# PR Review checklist
